composer.json

```diff
@@ -5,6 +5,7 @@
     "homepage": "https://www.ably.com/",
     "require": {
         "php": "^7.2 || ^8.0",
+        "rybakit/msgpack": "^0.9.1",
```

Comment:
can we make it `>=minimum_version_needed`. If the downstream app uses `ably-php`, at the same time, using `rybakit/msgpack` as another dependency shouldn't conflict with the current dependency.

Reply:
You can check https://getcomposer.org/doc/articles/versions.md#tilde-version-range-. currently, this would not allow downstream versions to be 1.x.x ( for future versions )

Reply:
I understand. We can make it >=0.9.1